scripts/functions.py

```diff
@@ -323,24 +322,23 @@ def createFuelSet(countries, rnwTechs, mineTechs, csvPath, generateInternational
     #Hydrogen Fuels
     #hy2FuelList = getHY2fuels(countries)
 
-    #Append lists together and write to a csv
+    #Append lists together and write to a dataframe
     #outputFuels = rnwFuelList + minFuelList + elcFuelList + hy2FuelList
     outputFuels = rnwFuelList + minFuelList + elcFuelList
     dfOut = pd.DataFrame(outputFuels, columns=['VALUE'])
-    dfOut.to_csv(csvPath, index=False)
+    return dfOut
 
-def createTechnologySet(countries, techsMaster, mineTechs, rnwTechs, trnTechsCsvPath, outputCsvPath, generateInternational):
-    # PURPOSE: Appends all technology name lists together and writes them to a CSV
+def createTechDataframe(countries, techsMaster, mineTechs, rnwTechs, trnTechsCsvPath, generateInternational):
+    # PURPOSE: Appends all technology name lists together and returns them as a CSV dataframe
     # INPUT:   countries = Dictionary holding countries as the key
     #                      and subregion as the values in a list
     #          techsMaster = List of the technologies to print over for getPWRtechs
     #          mineTechs = List of the technologies to print over for getMINtechs
     #          rnwTechs = List of the technologies to print over for getRNWtechs
-    #          trnTechsCsvPath = Trade csv datafile location
-    #          outputCsvPath = Path for the output
+    #          trnTechsCsvPath = Trade csv dataframe location
     #          generateInternational = True/False for whether function should
     #                                  create all international mining techs
-    # OUTPUT:  outputTechs = All technology lists appended together
+    # OUTPUT:  dfOut = fuel set dataframe

```

Comment:
Outputs a fuel set or a tech set dataframe? Name of function and description are different